hivemind/dht/__init__.py

```diff
@@ -155,6 +155,7 @@ async def _run():
                 method, args, kwargs = await loop.run_in_executor(pipe_awaiter, self._pipe.recv)
                 asyncio.create_task(getattr(self, method)(node, *args, **kwargs))
 
+        logger.info(f"Running DHT node on port {self.port}, initial peers = {self.initial_peers}")
```

Comment:
please move this inside _run AFTER the node is created. Otherwise the port is undefined.

Reply:
Moved the logger call back